Source/WebCore/page/Quirks.cpp

```diff
@@ -922,6 +922,13 @@ bool Quirks::shouldEnableEnumerateDeviceQuirk() const
 }
 #endif
 
+#if ENABLE(WEB_RTC)
+bool Quirks::shouldEnableRTCEncodedStreamsQuirk() const
+{
+    return needsQuirks() && m_quirksData.shouldEnableRTCEncodedStreamsQuirk;
```

Comment:
It seems like you are missing a line in `void handleFacebookQuirks(...)` to activate the Quirk!

Reply:
Yes, I will do this in a follow up.